pkgs/by-name/fv/fvm/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  buildDartApplication,
+  clang,
+  pkg-config,
+  gtk3,
+  cmake,
+  fetchFromGitHub,
+  nix-update-script,
+  lib,
+}:
+
+buildDartApplication rec {
+  pname = "fvm";
+  version = "3.2.1";
+
+  src = fetchFromGitHub {
+    owner = "leoafarias";
+    repo = pname;
+    tag = version;
+    hash = "sha256-i7sJRBrS5qyW8uGlx+zg+wDxsxgmolTMcikHyOzv3Bs=";
+  };
+
+  pubspecLock = lib.importJSON ./pubspec.lock.json;
+
+  nativeBuildInputs = [
+    clang
+    pkg-config
+    gtk3
+    cmake
+  ];
+
+  shellHook = ''
```

Comment:
Why `shellHook`?

Reply:
I think you want to set `env.PKG_CONFIG_PATH` and `env.LD_LIBRARY_PATH` instead.

Reply:
It is very likely that once you add `gtk3` to `buildInputs`, this becomes unnecessary.